app/static/tests/unit/components/basic/basicApp.test.ts

```diff
@@ -1,45 +1,55 @@
 // Mock the import of third party packages to prevent errors
-jest.mock("plotly.js-basic-dist-min", () => ({}));
-jest.mock("../../../../src/app/components/help/MarkdownItImport.ts", () => {
-    // eslint-disable-next-line func-names
-    return function () {
-        return {
-            use: jest.fn().mockReturnValue({
-                renderer: { rules: {} },
-                render: jest.fn()
-            })
-        };
-    };
+vi.mock("plotly.js-basic-dist-min", () => {
+    return {
+        default: undefined,
+        update: undefined
+    }
+});
+vi.mock("../../../../src/components/help/MarkdownItImport.ts", () => {
+  class MarkDownItClass {
+      constructor() {
+          return {
+              use: vi.fn().mockReturnValue({
+                  renderer: { rules: {} },
+                  render: vi.fn()
+              })
+          }
+      }
+  };
+  return {
+      default: {
+          default: MarkDownItClass
+      }
+  }
 });
```

Comment:
If some of these bits are going to pop up several times, I wonder if it would be worth importing them from a shared module?

Reply:
thats a good idea im not sure if you actually can even extract out vi.mocks, they seem to care about hoisting and not capturing stuff from the env and so on but i think there must be some functionality in vitest! i have made a ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5980/vi.mock-in-shared-module will do it in a future pr